.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2.1
+
+orbs:
+  codecov: codecov/codecov@1.0.4
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15.1
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: npm install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+      - run: npm run lint
+
+      - run: npm test
```

Comment:
No need to run both `lint` and `test`... take a look at the scripts:
```json
    "test": "xo && mocha test/**/*.js",
    "lint": "xo"
```

Reply:
I removed xo from test because I felt test was already a little overloaded with code coverage but I can put it back if you want. (The code you've pasted above is from before this PR.)